### ISSUE TITLE
generate_tables.ipynb is broken?

### ISSUE THREAD

**Issue by yuki-inaho** [google-research/multinerf#8]
Thank you for your great work !

I noticed your scripts/generate_tables.ipynb doesn't render correctly on the .ipynb viewer on github (like the figure I attached).
There seems to be something wrong with the code. Sorry if I misunderstood that. thx 

![image](https://user-images.githubusercontent.com/40379474/183252209-d4c42b1b-ce82-41ce-9c32-3b784ef10664.png)

**Comment by jonbarron** [comment]
You are correct! Thanks for reporting this. I think it's fixed now, please let us know if you have any other issues.
